content/persistent-disks.md

```diff
@@ -3,7 +3,13 @@
 
 Instance groups may need to store persistent data.
 
-If you attach a persistent disk to a virtual machine and then stop, terminate, or delete the VM, your persistent disk data remains intact. Attaching the persistent disk to another VM allows you to access your data.
+If you attach a persistent disk to a virtual machine and then stop the VM (such as `bosh delete-vm`), your
```

Comment:
"stop" and "bosh delete-vm" aren't really the same thing here. `bosh stop` would likely be more what operators perceive as "stopped". 

Perhaps something like
> If you attach a persistent disk to a virtual machine and then delete the VM via `bosh delete-vm`, your persistent disk data remains intact.